documentation-site/pages/_app.js

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+import React from 'react';
+import App, {Container} from 'next/app';
+
+import {Provider as StyletronProvider} from 'styletron-react';
+import {LightTheme, ThemeProvider} from 'baseui';
+
+import getStyletron from '../helpers/styletron';
+import Head from '../components/meta';
+
+export default class MyApp extends App {
+  static async getInitialProps({Component, router, ctx}) {
```

Comment:
Can this method be left out since unchanged?

Reply:
oh yeah, you are right, good catch!